src/Resources/translations/SonataAdminBundle.nl.xlf

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You should remove this file I think.

Reply:
It's a translation of SonataAdminBundle.en.xlf and SonataAdminBundle.de.xlf, If needed I will happily remove it.

Reply:
Oh, indeed.

But I think this was maybe a mistake and should be in the `xliff` file instead.
See in another language: https://github.com/sonata-project/SonataMediaBundle/blob/3.x/src/Resources/translations/SonataMediaBundle.fr.xliff#L17

And this translation is already in your `xliff` file 

Maybe @core23 can help about this since he made the PR: https://github.com/sonata-project/SonataMediaBundle/commit/da2d8a79eeecea292f2957a9444f350774c2f413

Reply:
Yes please 👍🏻